docs/react-samples/src/App.tsx

```diff
@@ -32,8 +26,29 @@ function App() {
   return (
     <>
       <h1>Contact Center widgets in a react app</h1>
-      <StationLogin  onLogin={onLogin} onLogout={onLogout} />
-      <UserState />
+      <input
+        type="text"
+        placeholder="Enter your access token"
+        value={accessToken}
+        onChange={(e) => setAccessToken(e.target.value)}
+      />
+      <button
+        disabled={accessToken.trim() === ""}
+        onClick={() => {
+          store.init({webexConfig, access_token: accessToken}).then(() => {
+            setIsSdkReady(true);
+          });
+        }}
+      >Init Widgets</button>
+      {/* write code to check if sdk is ready and load components */}
+      {
+        isSdkReady && (
+          <>
+            <StationLogin  onLogin={onLogin} onLogout={onLogout} />
+            <UserState />
+          </>
+        )
+      }
```

Comment:
I don't think this is possible. In the samples app, we cant pass our custom Webex object so we can remove this. We should definitely test this. I'm not sure how we will keep this in check in future.

Reply:
We can remove the isSDKReady state then.

Reply:
Could you please mention what is not possible?

If you look at the button onclick, you'll understand what's going on here. The `isSdkReady` is a state that's set after the `store.init` promise resolves. This is just to ensure the components load only after the store is initialised

Reply:
My bad. I got confused. I thought in the samples we are getting the webex object from devs. Thats what I thought is not possible. This is the flow where we get the access token. Resolving this.